### PR TITLE
Add unit test for mailchimp Authenticator

### PR DESCRIPTION
**Pull request by nalakas**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: component/src/test/java/org/wso2/carbon/identity/authenticator/mailChimp/MailChimpAuthenticatorTest.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
+package org.wso2.carbon.identity.authenticator.mailChimp;
```
Comment (kesavany) [inline component/src/test/java/org/wso2/carbon/identity/authenticator/mailChimp/MailChimpAuthenticatorTest.java]: Add License header.

File: component/src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (kesavany) [inline component/src/test/resources/testng.xml]: Change the year in the liecense header.

File: component/src/test/java/org/wso2/carbon/identity/authenticator/mailChimp/MailChimpAuthenticatorTest.java (hunk 1 above)
Comment (nalakas) [reply component/src/test/java/org/wso2/carbon/identity/authenticator/mailChimp/MailChimpAuthenticatorTest.java]: 7df79c7

File: component/src/test/resources/testng.xml (hunk 2 above)
Comment (nalakas) [reply component/src/test/resources/testng.xml]: fixed in 4bf5d3a

**Pull request merged**